samplr/README.md

```diff
@@ -51,3 +51,181 @@ e.g.
 ```bash
 samplrctl snippets list /tmp/local-repository
 ```
+
+## Testing
+
+To test the `samplr` code `cd` into the `samplr` directory and run
+
+```bash
+go test -v -race ./...
+```
+
+This will run the tests in the `samplr` directory as well as all subdirectories in it, recursively.
+
+### Integration Tests
+
+The integration tests take a bit longer to run, and require network access, so they require a special
+command line option to run: `-tags integration`
+
+```bash
+go test -v -tags integration ./...
+```
+
+This uses the Go `tag` feature, which is enabled by putting a comment in a file of form
+
+```go
+// +build TAGNAME
+```
+
+## Deploying
+
+The `Makefile` has several variables that can be overridden via environment variables
+in order to customize how the deployment occurs.
+
+```makefile
+# The (gcloud) test cluster that is being worked against
+GCP_CLUSTER_NAME ?= devrel-services
+GCP_CLUSTER_ZONE ?= us-central1-a
+# The service account to run as
+SERVICE_ACCOUNT_SECRET_NAME ?= service-account-maintnerd
+# Bucket settings for Repositories
+GCS_BUCKET_NAME ?= devrel-dev-settings
+REPOS_FILE_NAME ?= public_repos.json
+```
+
+These defaults are largely the same for dev and prod, but there are noteable differences
+
+---
+> __**Important!**__ After Deploying, it is necessary to find the container that is responsible for
+running googleapis/google-cloud-java and manually editing the Memory Requests and Limits to be
+3.5G and 4G respectively. This Repository has much higher memory requirements than all others.
+---
+
+### Deploy to DEV
+
+```bash
+# Ensure you are using the dev project
+make deploy
```

Comment:
The Makefile looks like it now uses PROD variables by default. Perhaps you meant to have the Makefile default to DEV values so that deploying to DEV can just be `make deploy`?

Reply:
I'm slightly confused. The Makefile has defaults for `dev` now (`devrel-dev-cluster` and `devrel-dev-settings`) were you looking at an outdated file or am I missing something?

Reply:
I meant the `makefile` code that's embedded in the README, that's what looks wrong.